src/main/java/com/google/devtools/build/lib/worker/WorkerMultiplexer.java

```diff
@@ -0,0 +1,283 @@
+// Copyright 2018 The Bazel Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.devtools.build.lib.worker;
+
+import com.google.devtools.build.lib.shell.Subprocess;
+import com.google.devtools.build.lib.shell.SubprocessBuilder;
+import com.google.devtools.build.lib.worker.WorkerProtocol.WorkResponse;
+import com.google.devtools.build.lib.vfs.Path;
+import java.io.File;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.Semaphore;
+
+/**
+ * An intermediate worker that receives response from the worker processes
```

Comment:
nit: Add missing trailing period, and this docstring fits in a single line.

Reply:
For the second part, I guess you meant to use `//` instead. While I see some single-line docstrings do it the same way like this
https://github.com/bazelbuild/bazel/blob/a24b5f389e54d631cfc91ae55b4001e01aa38931/src/main/java/com/google/devtools/build/lib/bazel/SpawnLogModule.java#L29
I just want to make sure what is the proper way. Thanks.

Reply:
No, I just meant:

```
/** Single-line docstring. */
```

Reply:
Fixed the period.